discovery_engine_core/web-api/src/bin/ingestion.rs

```diff
@@ -80,21 +89,13 @@ struct IngestedDocument {
 #[derive(Debug, Clone, Serialize)]
 struct IngestionError {
     /// List of Document Indices which were not successfully processed
-    documents: Vec<ErroredDocumentId>,
-}
-
-#[derive(Serialize, Debug, Clone)]
-pub struct ErroredDocumentId {
```

Comment:
I think we need this wrapper, because without it `IngestionError` is serialized to this:
```json
{
  "documents": [
    "doc_id_1",
    "doc_id_2",
    "doc_id_n",
  ]
}
```

and the spec says is should be an array of objects with "id" as property
```json
{
  "documents": [
    { "id": "doc_id_1" },
    { "id": "doc_id_2" },
    { "id": "doc_id_n" }
  ]
}
```

Reply:
fixed. i also renamed it a bit to signal that the container can potentially be extended to contain more info about the failed doc than just the id.